test/metabase/query_processor/middleware/catch_exceptions_test.clj

```diff
@@ -111,11 +112,11 @@
                                                  (context/raisef (ex-info "Something went wrong."
                                                                    {:query-execution {:a            100
                                                                                       :b            200
+                                                                                      :card_id      300
                                                                                       ;; these keys should all get removed
-                                                                                      :result_rows  300
-                                                                                      :hash         400
+                                                                                      :result_rows  400
```

Comment:
i understand the presence of `card_id` here, but why did the `result_rows` value change?

Reply:
these are completely arbitrary numeric values actually, so I changed it to keep it as a nice tidy sequence of 100's. too bad the other change added another 500, so this was pointless

Reply:
I see. I was surprised because i saw that there was a change expecting `:card_id 300` but no other changes. But i guess they were `schema=` checks so they still work? Either way it sounds like the change is benign.